.github/workflows/ci.yml

```diff
@@ -178,8 +405,8 @@ jobs:
           submodules: true
       - name: Install Python dependencies
         run: |
+          python3 -m pip install --upgrade pip setuptools wheel
```

Comment:
Do you remember if `wheel` is needed by RLTest as well? Because if I remember correctly, setuptools was needed by RLTest. I guess normally RLTest should declare dependency to these.

Reply:
I am not sure either of those is nadeed by RLTest (yet). This here is to solved a Python setup issue on macOS in particular, so not related to the tests and the project.

Reply:
Ok, got it. 

Btw, is there anything else you want to add to PR or should we merge it?

Reply:
I think we can merge it, but I have no rights to press the "merge" in this project.